README.md

```diff
@@ -3,54 +3,41 @@ Datashader
 
 [![Build Status](https://travis-ci.org/bokeh/datashader.svg)](https://travis-ci.org/bokeh/datashader)
 
-The Datashader project is focused on building better ways to visualize
-very large datasets, using intelligent server-side downsampling,
-automatic computations performed on the data as it progresses through
-the visualization pipeline, and other related techniques.  The project
-is under active development, and all the code and documentation is
+
+Datashader is a graphics pipeline system for creating meaningful
+representations of large amounts of data. It breaks the creation of images into
+2 steps:
+
+1. Aggregation
+
+   Data is binned per-pixel by various user specified methods into aggregates.
+
+2. Transformation
+
+   These aggregates are then further processed to create an image.
+
+Using this very general pipeline, many interesting data visualizations can be
+created in a performant and scalable way. Datashader contains tools for easily
+creating these pipelines in few lines of code.
```

Comment:
It's also important that the pieces of the pipeline are easily configurable and swappable, which is not true for standard viz approaches.